string robot_id
float64 xPos
float64 yPos
float64[] rotationQuaternion
float64 currentMicHeight
float64 currentMicAngle
# string currentLEDAnimation
# int[] currentLEDColour
# float64 currentFlashFrequency